GliaWidgets/Lib/View/View.swift

```diff
@@ -14,10 +14,26 @@ class View: UIView {
 
     init() {
         super.init(frame: .zero)
+        setup()
     }
 
     @available(*, unavailable)
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    override func updateConstraints() {
+        if needsDefineLayout {
+            needsDefineLayout = false
+            defineLayout()
+        }
+        super.updateConstraints()
+    }
+
+    func setup() {}
+    func defineLayout() {}
```

Comment:
I think previously we have named the function just "layout", perhaps it would be better to keep it consistent with other views?

Reply:
the main reason why I named it `defineLayout` is because this method contains mechanisms for defining the layout, so it calls once for defining layout but not for layouting in general. 
Does it make sense? or what do you think about naming @kuldar-daniel  @igorkravchenko ?

Reply:
I think `layout` method is common among all views in SDK. It serves the same purpose as `defineLayout` - defines layout. I think `defineLayout` is more meaningful name, but I think it makes sense to use `layout` for consistency as @kuldar-daniel suggested.